conceptual/Npgsql/release-notes/5.0.md

```diff
@@ -11,10 +11,10 @@ Npgsql 5.0 is out and available at nuget.org.
 
 The major new features of 5.0 are:
 
-*TO BE FILLED*
+* `NULL` values inside arrays of value types are now properly supported via arrays of `Nullable<T>`. Previously an attempt to read or write an array of nullable value types (e. g. `Nullable<Int32>[]`) resulted in an exception. Also `NULL` values in the database array contained the default value of the corresponding value type on the .Net side making them indistiguishable from real values (e. g. `ARRAY[0,1,NULL,3]` resulted in `Int32[]{0,1,0,3}`).
```

Comment:
`.Net -> .NET`

Reply:
`Int32` -> `int`
`Nullable<T>` -> `T?` (except maybe in the first sentence?)

(below as well)

Reply:
fixed